### PR TITLE
Docker Drone Installation Instructions

### PR DESCRIPTION
**Pull request by AR2100**
# Description
This PR does the following:
- Update docker install instructions 
- Fix  `Couldn't find directory for external packages...` and `Couldn't find .git folder in external package external/nanopb/.git ...` CMake errors

# Testing Steps
1. Follow docker installation instructions using manual build method (someone with a mac should try as well) 
3. `colcon build` urc drone and urc rover
4. check that gui works (try `gazebo` and `rviz2`)

Expectation: everything works

# Self Checklist
- [x] I have formatted my code using `ament_uncrustify --reformat`
- [x] I have tested that the new behavior works

### CODE REVIEW

**Comment by AR2100** [comment]
Once this is approved, I'll update Dockerhub repo

File: docker_gui/Dockerfile [hunk 1]
```diff
@@ -16,20 +16,27 @@ RUN apt-get update --fix-missing && apt-get -y install \
...
-# Clone git repo
-WORKDIR /colcon_ws/src
-RUN rm -rf urc-software
-RUN git clone https://github.com/RoboJackets/urc-software.git --recursive
-
-# Install our ROS dependecies
-WORKDIR /colcon_ws
+# Update ROS Packages
 RUN rosdep update
-RUN apt-get update --fix-missing
```
Comment (a-stickan) [inline docker_gui/Dockerfile:32]: Don't remove this, best practice imo

**Review by a-stickan — CHANGES_REQUESTED** [review]
Should be good assuming that one line is changed

**Comment by a-stickan** [comment]
Stale PR, closing

**Pull request closed without merging**